db/src/main/resources/db/migration/V1_045__create_identity_and_eligibility_outcome_column.sql

```diff
@@ -0,0 +1,46 @@
+alter table payment_cycle add column identity_and_eligibility_response jsonb;
+
+update payment_cycle set identity_and_eligibility_response = '{
+    "eligibilityStatus": "confirmed",
+    "deathVerificationFlag": "n/a",
+    "mobilePhoneMatch": "matched",
+    "emailAddressMatch": "matched",
+    "addressLine1Match": "matched",
+    "postcodeMatch": "matched",
+    "pregnantChildDOBMatch": "not_supplied",
+    "qualifyingBenefits": "universal_credit",
+    "householdIdentifier": "",
+    "dobOfChildrenUnder4": [],
```

Comment:
Strictly speaking we should probably use the list of children's dob from the claim, but as this is only used for reference I don't think we need to worry.